src/layout.rs

```diff
@@ -278,6 +308,36 @@ impl Layout {
                     self.do_action(action, coord, delay);
                 }
             }
+            Sequence { delay, actions } => {
+                for key_event in actions {
+                    match *key_event {
+                        SequenceEvent::Press(keycode) => {
+                            self.stacked.push_back(Event::SequencePress(keycode).into());
```

Comment:
why `.into()`?

Reply:
The `into()` is there because it would not compile without it.  It throws this error:

```
   Compiling keyberon v0.1.1 (/home/riskable/workspace/rust/embedded/keyberon)
error[E0308]: mismatched types
   --> /home/riskable/workspace/rust/embedded/keyberon/src/layout.rs:315:52
    |
315 | ...                   self.stacked.push_back(Event::SequencePress(keycode));
    |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `layout::Stacked`, found enum `layout::Event`
```